zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/SparkSQLGenerator.scala

```diff
@@ -0,0 +1,636 @@
+package edu.uci.ics.cloudberry.zion.model.impl
+
+import edu.uci.ics.cloudberry.zion.model.datastore.{IQLGenerator, IQLGeneratorFactory, QueryParsingException}
+import edu.uci.ics.cloudberry.zion.model.schema._
+import play.api.libs.json.Json
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+import scala.collection.mutable.ArrayBuffer
+
+
+class SparkSQLGenerator extends IQLGenerator {
+
+  /**
+    * represent the expression for a [[Field]]
+    *
+    * @param refExpr the expression for referring this field by the subsequent statements
+    * @param defExpr the expression the defines the field
+    *
+    */
+  case class FieldExpr(refExpr: String, defExpr: String)
+
+  /**
+    * Partial parse results after parsing each [[Statement]]
+    *
+    * @param strs    a sequence of parsed query strings, which would be composed together later.
+    * @param exprMap a new field expression map
+    */
+  case class ParsedResult(strs: Seq[String], exprMap: Map[String, FieldExpr])
+
+  //  protected val typeImpl: SparkSqlImpl
+
+  protected val sourceVar: String = "t"
+
+  protected val appendVar: String = "ta"
+
+  protected val unnestVar: String = "unnest"
+
+  protected val lookupVar: String = "l"
+
+  protected val groupVar: String = "*"
+
+  protected val groupedLookupVar: String = "ll"
+
+  protected val groupedLookupSourceVar: String = "tt"
+
+  protected val globalAggrVar: String = "*"
+
+  protected val outerSelectVar: String = "s"
+
+  protected val quote = "`"
+
+  protected val suffix: String = ";"
+
+  val datetime: String = ""
+  val round: String = "round"
+
+  val dayTimeDuration: String = "day_time_duration"
```

Comment:
Same problem here. If you do not implement or do not plan to implement these functions, remove them.